include/openthread/platform/time.h

```diff
@@ -52,17 +52,34 @@ extern "C" {
  */
 
 /**
- * Get the current time (64bits width).
+ * Get the current time in microseconds referenced to a continuous monotonic
+ * local radio clock (64 bits width).
+ *
+ * The radio clock SHALL not wrap during the device's uptime. Implementations
```

Comment:
In case of a normative requirement the sentence should be:

```suggestion
 * The radio clock SHALL NOT wrap during the device's uptime. Implementations
```

Reply:
Fixed.